tests/test_dashboard.py

```diff
@@ -12,31 +13,56 @@
 from atmo.views import server_error
 
 
-@pytest.fixture
-def dashboard_spark_jobs(now, test_user):
-    for x in range(10):
-        identifier = 'test-spark-job-%s' % x
+LAST_SPARK_JOB_ID = 0
+
+
+def create_spark_jobs(num, date, owner):
+    global LAST_SPARK_JOB_ID
```

Comment:
It's still relevant since I'm calling `create_spark_jobs` twice and passing in two different users. Without it the spark job identifier will create clashes since it is required to be unique.